File: src/main/java/um/tesoreria/facturador/model/dto/FacultadDto.java
package um.tesoreria.facturador.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class FacultadDto {

    private Integer facultadId;
    private String nombre = "";
    private String codigoempresa = "";
    private String server = "";
    private String dbadm = "";
    private String dsn = "";
    private BigDecimal cuentacontable = BigDecimal.ZERO;
    private String apiserver = "";
    private Long apiport = 0L;

}

File: src/main/java/um/tesoreria/facturador/kotlin/tesoreria/core/dto/FacultadDto.kt
package um.tesoreria.facturador.kotlin.tesoreria.core.dto

import java.math.BigDecimal

data class FacultadDto(

    var facultadId: Int? = null,
    var nombre: String = "",
    var codigoempresa: String = "",
    var server: String = "",
    var dbadm: String = "",
    var dsn: String = "",
    var cuentacontable: BigDecimal = BigDecimal.ZERO,
    var apiserver: String = "",
    var apiport: Long = 0L

)